.circleci/config.yml

```diff
@@ -677,6 +677,23 @@ jobs:
             cd packages/celotool
             ./ci_test_validator_order.sh checkout ${CELO_BLOCKCHAIN_BRANCH_TO_TEST}
 
+  end-to-end-cip35-test:
```

Comment:
Even if I know that the cip35 explains everything, maybe we could add a little comment, or change the `name` to instead of saying "Run test" we could read something like, "Testing ethereum Txs", mostly because this will be shown in circle, and could be too obfuscated for someone new.

Reply:
Sounds good, I renamed it to `end-to-end-cip35-eth-compatibility-test`